api/account.go

```diff
@@ -43,3 +49,39 @@ func (api *API) ListVpcs() ([]map[string]interface{}, error) {
 
 	return data, nil
 }
+
+func (api *API) RotatePassword(instanceID int) error {
+	var (
+		failed map[string]interface{}
+		path   = fmt.Sprintf("api/instances/%d/account/rotate-password", instanceID)
+	)
+
+	response, err := api.sling.New().Post(path).Receive(nil, &failed)
+	if err != nil {
+		return err
+	}
+	if response.StatusCode != 204 {
```

Comment:
Might make sense to future proof here and allow response code 200 as success too

I recall someone asking for the password/key to be part of the reply when the rotate, and "204 No Content" wouldn't be suitable if we started doing that.

Reply:
Any idea on how we want the response body if we choose to add password/apikey in the response?

Reply:
Heh :) Now I get what you talked about.